### PR TITLE
User send request service

### PR DESCRIPTION
**Pull request by devtuananh**
https://dev.framgia.com/issues/184356

### CODE REVIEW

**Comment by devtuananh** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -210,6 +210,10 @@ textarea {
   width: 100%;
 }
 
+.image-rooms-home {
+  margin: 5px 5px 5px 5px;
```
Comment (huongnguyenmta) [inline app/assets/stylesheets/custom.scss]: => `margin: 5px`

File: app/views/bookings/_booking.html.erb [hunk 2]
```diff
@@ -13,6 +13,14 @@
       <i class="fa fa-caret-right"></i>
       <%= t ".end_time", time: l(booking.end_time, format: :own) %>
     </a></li>
+    <% if booking.status == "accepted" %>
```
Comment (huongnguyenmta) [inline app/views/bookings/_booking.html.erb]: dùng enum

**Comment by huongnguyenmta** [comment]
commented

**Comment by devtuananh** [comment]
ready

**Pull request merged**